.github/workflows/build-samples.yml

```diff
@@ -11,4 +11,4 @@ jobs:
   call-workflow-passing-data:
     uses: ./.github/workflows/test-workflow.yml
     with:
-      test-path: ./Scripts/test_samples
```

Comment:
We don't have any test files for samples, right? I'm assuming that's why you changed this?